packages/cards/src/TemplateRowCard.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Avatar,
+  Badge,
+  Box,
+  Flex,
+  HStack,
+  Heading,
+  Icon,
+  Image,
+  Link,
+  Spacer,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { SettingsMenuButton, SettingsMenuButtonProps } from '@easyfeedback/buttons'
+import dayjs from 'dayjs'
+import { IoTimerOutline } from 'react-icons/io5'
+
+import { ViewsStats, ViewsStatsProps } from './utils'
+
+const getFormattedDate = (date: string) => dayjs(date).format('DD.MM.YYYY | HH:mm')
+
+type Creator = { name: string; imageSrc?: string }
+
+export type TemplateRowCardProps = {
+  /** The hyperlink to the template. */
+  href: string
+  /** The title of the template. */
+  title: string
+  /** The image src attribute. */
+  imageSrc: string
+  /** All the tags to display. */
+  tags?: string[]
+  /** The time of the last edit. */
+  editedAt: string
+  /** The starting time of the survey. */
+  startTime: string
+  /** The ending time of the survey. */
+  endTime?: string
+  /** The creator of the template. */
+  creator?: Creator
+} & Pick<SettingsMenuButtonProps, 'menuItems'> &
+  Pick<ViewsStatsProps, 'views'>
+
+// TODO: Add mobile styles after the designer added them
+
+/**
+ * `TemplateRowCard` is a component for displaying the informations, stats and preview of the
+ * available templates.
+ */
+export const TemplateRowCard = ({
+  href,
+  title,
+  imageSrc,
+  tags,
+  editedAt,
+  startTime,
+  endTime,
+  creator,
+  views,
+  menuItems,
+}: TemplateRowCardProps) => {
+  return (
+    <Flex
+      p="2"
+      borderWidth="1px"
+      borderColor="gray.300"
+      borderRadius="base"
+      w="full"
+      data-testid="TemplateRowCard"
+    >
+      <HStack spacing="6">
+        <Link href={href} title={title}>
+          <Image
+            src={imageSrc}
+            alt={title}
+            boxSize="3.5rem"
+            objectFit="cover"
+            borderRadius="base"
+          />
+        </Link>
+
+        <VStack alignItems="flex-start">
+          <HStack>
+            <Heading size="sm">
+              <Link href={href}>{title}</Link>
+            </Heading>
+
+            {tags && (
+              <HStack data-testid="tags">
+                {tags.map((tag, index) => (
```

Comment:
unlimited tags?
I will drop a commit with the storybook arguments to check it later. it may break the layout

Reply:
As long as there no other information: YES.
This is also a point like the mobile view we should keep in mind and work on later.